### DISCUSSION TITLE
Poll answer is not shown selected although it was answered

### DISCUSSION THREAD

**Discussion by HarHarLinks** [element-hq/element-meta#733] [Ideas]
### Steps to reproduce

yesterday:

1. Send a Poll with E desktop stable w10
2. answer it from E desktop stable w10

today:

3. look at it from E desktop nightly linux

#### What did you expect?
poll shows vote

#### What happened instead?
no votes cast

![image](https://user-images.githubusercontent.com/2803622/153253710-b1fb9398-8db0-4e67-9902-955fb49f494a.png)

although the response clearly exists

### Operating system

arch

### Application version

Element Nightly version: 2022020901 Olm version: 3.2.8

### How did you install the app?

aur

### Homeserver

1.51

### Will you send logs?

No

**Comment by HarHarLinks** [comment]
I'm still on the same version, but someone else has now answered and it is showing both votes.

**Comment by HarHarLinks** [comment]
following screenshots were taken at the same time:

timeline:
![image](https://user-images.githubusercontent.com/2803622/153412842-d2c92fa0-a6ef-48ba-b2ba-fb7a91fd6f5e.png)

android:
![image](https://user-images.githubusercontent.com/2803622/153417779-08fcb3b4-f2b4-4a32-90dc-60a629400059.png)

pins (desktop labs):
![image](https://user-images.githubusercontent.com/2803622/153412864-f7369a2f-d2e4-44b9-be04-80aa0b2aeec8.png)

the poll event is publicly accessibly at: https://matrix.to/#/!cNSQwPqhHKkIZdBnvt:devture.com/$H82vXTJjjHTFSEK0rjmGH9LlNwHAR4rQjaIb8mN8xPE?via=devture.com&via=matrix.org&via=schnuffle.de

**Comment by waclaw66** [comment]
This happens to me regularly, it shows just votes counter, but no votes.
Element Web 1.10.11

**Comment by mnaturel** [comment]
Hi @HarHarLinks, thanks for raising this issue. Not seeing the results until the user votes was done by design. I changed it to en enhancement issue. If you feel there is still a bug with the current implementation, please add more comments.

**Reply by HarHarLinks** [reply]
> Not seeing the results until the user votes was done by design.

That is not the issue here, this is an implementation bug in element web.

I voted on one device, yet it wouldn't show that I voted on another (desktop) device.

**Reply by kerryarchibald** [reply]
No longer reproduces with Element ios 1.9.7 and latest nightly desktop on mac os.
@HarHarLinks can you confirm this is still reproducible for you?

**Reply by HarHarLinks** [reply]
this particular poll works fine now, when you scroll around enough so element sync all relevant vote events (we need backwards-linked relations, I think there's an MSC.)

it's fine to close this, except it doesn't belong here in the "feature suggestions" discussions.
